### PR TITLE
Pygame cheat sheet

### PR DESCRIPTION
**Pull request by amarlearning**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New Pygame Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes
What changes does this PR introduce? 
New Goodie: Python Cheat Sheet

###### Related Issues or Discussions 
What issues or discussions (if any) does this PR address? Please provide the issue # or link.

###### People to notify
@pjhampton 

---

Instant Answer Page: https://duck.co/ia/view/pygame_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention

### CODE REVIEW

**Comment by daxtheduck** [comment]
@amarlearning Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by tagawa** [comment]
Hi @amarlearning. Thank you for contributing!

I notice there are two files in the commit. Please could you remove `share/goodie/cheat_sheets/json/htaccess.json` from the pull request?

File: share/goodie/cheat_sheets/json/pygame.json [hunk 1]
```diff
@@ -0,0 +1,131 @@
+{
+    "id": "pygame_cheat_sheet",
+    "name": "pygame",
+    "description": "A quick reference to speed on the Pygame 2D game framework",
```
Comment (pjhampton) [inline share/goodie/cheat_sheets/json/pygame.json]: A ~~quick~~ reference ~~to speed~~ on the Pygame 2D game framework

File: share/goodie/cheat_sheets/json/pygame.json [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+            },
+            {
+                "key": "screen = pygame.display.set_mode((width, height))",
+                "val": " Initializes and creates the window where your game will run.'"
+            }
+        ],
+        "Surfaces, Images, and Transformations": [
+            {
+                "key": "Surface.fill(color)",
+                "val": "Fills surface with a solid color. Argument is a tuple of RGB values.\ne.g. (255,0,255)."
```
Comment (pjhampton) [inline share/goodie/cheat_sheets/json/pygame.json]: _The_ argument

File: share/goodie/cheat_sheets/json/pygame.json [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+                "key": "Surface.convert()",
+                "val": "Changes pixel format of the Surface’s image to the format used by the main display."
+            },
+            {
+                "key": "Surface.convert_alpha()",
+                "val": "Same as above, but when the Surface’s image has alpha transparency values to deal with."
+            },
+            {
+                "key": "Surface.get_rect()",
+                "val": " Returns a Rect that will tell you the dimensions and location of the surface."
```
Comment (pjhampton) [inline share/goodie/cheat_sheets/json/pygame.json]: Space before "Returns"

File: share/goodie/cheat_sheets/json/pygame.json [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+                "key": "pygame.image.load(filename)",
+                "val": "Loads image from disk and returns a Surface."
+            },
+            {
+                "key": "pygame.transform.rotate(Surface, angle)",
+                "val": "Rotates Surface counterclockwise by degrees."
+            },
+            {
+                "key": "pygame.transform.scale(Surface, (width, height))",
+                "val": " Resizes Surface to new resolution."
```
Comment (pjhampton) [inline share/goodie/cheat_sheets/json/pygame.json]: same here

**Pull request closed without merging**